src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/notifications/NotificationFactory.java

```diff
@@ -0,0 +1,41 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.notifications;
+
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.elements.mercury.components.discussions.common.PostEntity;
+
+public class NotificationFactory {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1118)